static/images/info_alt.svg

```diff
@@ -1,9 +0,0 @@
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg"
```

Comment:
No need for this SVG file -- we can call any SLDS icons directly by `category` and `name`: https://react.lightningdesignsystem.com/components/icons/